### PR TITLE
Rename "Think something" to "Emote", make it more freeform

### PR DESCRIPTION
**Pull request by KittyTac**
#### Summary
Balance "Rename 'Think something' to 'Emote', make it more freeform"

#### Purpose of change
While #61490 added the ability to think something to yourself, giving a message with no effect (which really helps roleplaying), it felt somewhat limited. Why not allow adding arbitrary flavor messages to the log?

#### Describe the solution
Change a const name and some lines of text, remove the "You think" prepended to the message.

#### Describe alternatives you've considered
1. Obligatory leaving as is.
2. Keeping "You think", or "You", as a separate option? _I'd like feedback on this one._

#### Testing
I'm currently unable to compile the game myself, but as this is a very minor change I don't think testing is necessary here.

#### Additional context
Yes, you can technically "do" things with this that you physically can't, but it's the same for "Think something". E.g you can already do `You think for a moment and blow up the horde with a fireball!`. You'd have to go out of your way to immersion-break that way, so I think it's fine.

### CODE REVIEW

**Comment by KeremBabaG** [comment]
I think it should stay as it is. With "you think" you could do a inner monologue/constant stream of consciousnes type of thing, "flavor message" sounds cheesy and immersion breaking.

**Comment by KittyTac** [comment]
> I think it should stay as it is. With "you think" you could do a inner monologue/constant stream of consciousnes type of thing, "flavor message" sounds cheesy and immersion breaking.

The thing is, you could do that with a flavor message too. Nothing is stopping you from starting it with `You think`. And with it you can do something like `Looking back at the pile of dead zombies you left behind, you twirl the still-smoking pistol in your hand.`

If your problem is with the name, I don't know what to rename it to that suggests a meaningless free action.

**Comment by KeremBabaG** [comment]
> > I think it should stay as it is. With "you think" you could do a inner monologue/constant stream of consciousnes type of thing, "flavor message" sounds cheesy and immersion breaking.
> 
> The thing is, you could do that with a flavor message too. Nothing is stopping you from starting it with `You think`. And with it you can do something like `Looking back at the pile of dead zombies you left behind, you twirl the still-smoking pistol in your hand.`
> 
> If your problem is with the name, I don't know what to rename it to that suggests a meaningless free action.

Yeah you're right. nvm

**Comment by ashGlaw** [comment]
I added the thinking option in the first place, and I wanted to let you know that I'm down with this. Making it a flavor action is pretty easy and opens up other possibilities. I'm still considering adding the option to speak a sentence out loud, perhaps, with a small volume instead of a shout, But this may supersede that, because you can throw quotes up and say something that people won't react to. Which is mostly fine. This works for me 👍

**Comment by KittyTac** [comment]
I think the checks are broken, I'll close and reopen.

**Comment by KittyTac** [comment]
The only failing tests are about starting items which this in no way affects, so this is ready to merge.

**Comment by ashGlaw** [comment]
Looking forward to it

**Comment by KittyTac** [comment]
Renamed as per suggestion by @Maleclypse to make it less fourth-wall-breaking?

File: src/npctalk.cpp [hunk 1]
```diff
@@ -878,7 +878,7 @@ void game::chat()
 
     nmenu.addentry( NPC_CHAT_YELL, true, 'a', _( "Yell" ) );
     nmenu.addentry( NPC_CHAT_SENTENCE, true, 'b', _( "Yell a sentence" ) );
-    nmenu.addentry( NPC_CHAT_THINK, true, 'T', _( "Think something" ) );
+    nmenu.addentry( NPC_CHAT_MISC, true, 'T', _( "Miscellaneous action" ) );
```
Comment (Fris0uman) [inline src/npctalk.cpp:881]: `Miscellaneous action` seems misleading, this looks like it's going to actually do something

Comment (KittyTac) [reply src/npctalk.cpp:881]: > `Miscellaneous action` seems misleading, this looks like it's going to actually do something

In the blurb that shows up in the textbox it says it'll have no in-game effect. What would be a good name that doesn't break the fourth wall?

Comment (fairyarmadillo) [reply src/npctalk.cpp:881]: 'Emote' is commonly used in games to represent fluff actions.

Comment (KittyTac) [reply src/npctalk.cpp:881]: > 'Emote' is commonly used in games to represent fluff actions.

PERFECT! I'll do that.

**Pull request merged**